### PR TITLE
Env var cleanup

### PR DESCRIPTION
**Pull request by eessex**
Additional updates related to https://artsyproduct.atlassian.net/browse/PLATFORM-737

### CODE REVIEW

File: src/desktop/config.coffee [hunk 1]
```diff
@@ -119,14 +120,14 @@ module.exports =
   STRIPE_PUBLISHABLE_KEY: null
   TEAM_BLOGS: '^\/life-at-artsy$|^\/artsy-education$|^\/buying-with-artsy$'
   TARGET_CAMPAIGN_URL: '/seattle-art-fair-2017'
-  TRACK_PAGELOAD_PATHS: null
+  TRACK_PAGELOAD_PATHS: "home|collect|artist|artwork|show|article|feature|gene|tag|partner|fair|news|new-artwork"
```
Comment (joeyAghion) [inline src/desktop/config.coffee]: This risks creating error or warning noise in environments like development and test when Volley (which receives these metrics and sends them to statsd) isn't available.

File: src/mobile/config.coffee [hunk 2]
```diff
@@ -57,8 +57,7 @@ module.exports =
   SHOW_ANALYTICS_CALLS: false
   STRIPE_PUBLISHABLE_KEY: null
   TARGET_CAMPAIGN_URL: '/seattle-art-fair-2017'
-  TRACK_PAGELOAD_PATHS: null
-  TWITTER_CONSUMER_KEY: null
+  TRACK_PAGELOAD_PATHS: "home|collect|artist|artwork|show|article|feature|gene|tag|partner|fair|news|new-artwork"
```
Comment (joeyAghion) [inline src/mobile/config.coffee]: ditto

**Comment by joeyAghion** [comment]
Tests succeeded on retry. 🤷‍♂

**Comment by eessex** [comment]
This has been updated to restore `TRACK_PAGELOAD_PATHS`

**Pull request merged**